lib/nostrum/shard/dispatch.ex

```diff
@@ -111,8 +113,15 @@ defmodule Nostrum.Shard.Dispatch do
       :ets.insert(:channel_guild_map, {channel.id, guild.id})
     end)
 
-    if guild.member_count >= @large_threshold and
-         Application.get_env(:nostrum, :request_guild_members, false) do
+    has_members = (Intents.get_enabled_intents() &&& 1 <<< 1) == 1 <<< 1
```

Comment:
Do you think we should create some helper in the `Intents` module to prevent having to include these messy details whenever we want to check intents?

Reply:
Resolved with introduction of `Intents.has_intent?/1` in 08e886864b38bb2f096e486abf328c6d00374f0e